var/spack/repos/builtin/packages/msmpi/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Msmpi(Package):
+    """A Windows-specced build of MPICH provided directly by
+    Microsoft Support Team
+    """
+
+    homepage = "https://www.microsoft.com/en-us/download/default.aspx"
+    maintainers = ["jpopelar"]
+
+    version(
+        "10.0",
+        sha256="7dae13797627726f67fab9c1d251aec2df9ecd25939984645ec05748bdffd396",
+        extension="exe",
```

Comment:
Does this work? AFAIK this should fail because we'll still try to extract the extension and fail (because it's a binary). Should this be set to `expand=False`?